src/main/java/io/jenkins/plugins/checks/github/GitHubChecksAction.java

```diff
@@ -0,0 +1,39 @@
+package io.jenkins.plugins.checks.github;
+
+import hudson.model.InvisibleAction;
+import io.jenkins.plugins.checks.api.ChecksConclusion;
+
+public class GitHubChecksAction extends InvisibleAction {
+
+    private final long id;
+    private final String name;
+    private final ChecksConclusion conclusion;
```

Comment:
Maybe we can attach the whole `GitHubCheksDetails` here? No idea yet.

Reply:
Yes, I wasn't sure on the serialisation of actions - I thought best to keep it simple in the first instance with just simple types.